src/pages/Auth/ResetPasswordPage.jsx

```diff
@@ -1,10 +1,97 @@
+import React, {useState} from 'react'
+
```

Comment:
Remove this line. Group all external library/components imports together and custom files imports together. Have space only between these 2 groups.